app/controllers/questions/start_controller.rb

```diff
@@ -1,6 +1,8 @@
 module Questions
   class StartController < ApplicationController
     def index
+      reset_session
```

Comment:
Do we want to reset the whole session, or just the selected answers?

Reply:
Yes, should be the whole session - no defined need for anything more granular at this point.

Reply:
@lexijax My thought was that I'm not sure what else gets stored in the session (the principle being don't mess with what you don't control) and that this might have implications for analytics or rate limiting. I don't know that though.

Reply:
@dgmstuart `reset_session` only deletes the cookie that Rails uses. Analytics etc will use their own separate cookies.

Reply:
@leeky Indeed. My thought was that maybe hotjar uses it to identify unique users. Probably not